src/components/Comments/Comment.js

```diff
@@ -128,14 +133,14 @@ const Comment = ({
   };
 
   const username = comment?.data?.anon_user_info_color_name
-    ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+    ? `${comment.data.anon_user_info_color_name} ${comment.data?.anon_user_info_emoji_name}`
     : user?.data?.username;
 
   const onPressDM = async () => {
     try {
       setLoading({...loading, loadingDm: true});
       if (!comment?.data?.anon_user_info_color_name) {
-        const channelName = [username, profile?.myProfile?.username].join(',');
+        const channelName = username;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The username generation logic has been changed. Make sure this change doesn't affect other parts of the code where the username is used.

```diff
- const username = comment?.data?.anon_user_info_color_name
-    ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
-     : user?.data?.username;
+ const username = comment?.data?.anon_user_info_color_name
+     ? `${comment.data.anon_user_info_color_name} ${comment.data?.anon_user_info_emoji_name}`
+      : user?.data?.username;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->